clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/actions/PrepareTitusDeploy.java

```diff
@@ -285,6 +290,13 @@ private void mergeSourceDetailsIntoDescription(
         .getResources()
         .setNetworkMbps(
             orDefault(description.getResources().getNetworkMbps(), sourceJob.getNetworkMbps()));
+    description
+        .getResources()
+        .setSignedAddressAllocations(
+            orDefault(
+                description.getResources().getSignedAddressAllocations(),
+                sourceJob.getSignedAddressAllocations()));
```

Comment:
might be worth a side comment saying that if the request does not include the allocations, we would fall back on the source job